adoc/admin-troubleshooting-etcd.adoc

```diff
@@ -0,0 +1,136 @@
+= ETCD Troubleshooting
+
+== Introduction
+
+This document aims to describe debugging a {etcd} cluster.
+
+The required etcd logs are part of the `supportconfig`, a utility that collects are the required information for debugging a problem. The rest of the document provides information on how you can obtain these information manually.
+
+== ETCD container
+
+ETCD is a distributed reliable key-value store for the most critical data of a distributed system. It is running **only on the master** nodes in a form a container application. For instance, in a cluster with 3 master nodes, it is expected
+to have 3 etcd instances as well:
+
+[source,bash]
+----
+kubectl get pods -n kube-system -l component=etcdNAME
```

Comment:
Ouch, missing break line here between `etcd` and `NAME`

Reply:
Thanks for spotting this!